<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeyboardConfigWidget</class>
 <widget class="QWidget" name="KeyboardConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>411</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="numlockGroupBox">
     <property name="whatsThis">
      <string>If supported, this option allows you to setup the state of NumLock after Plasma startup.&lt;p&gt;You can configure NumLock to be turned on or off, or configure Plasma not to set NumLock state.</string>
     </property>
     <property name="title">
      <string>NumLock on Plasma Startup</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="0" column="0">
       <widget class="QGroupBox" name="numlockButtonGroup">
        <property name="flat">
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <widget class="QRadioButton" name="radioButton1">
           <property name="text">
            <string>T&amp;urn on</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="radioButton2">
           <property name="text">
            <string>&amp;Turn off</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QRadioButton" name="radioButton3">
           <property name="text">
            <string>Leave unchan&amp;ged</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="keyboardRepeatGroupBox">
     <property name="title">
      <string>Keyboard Repeat</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QGroupBox" name="keyboardRepeatButtonGroup">
        <property name="flat">
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QRadioButton" name="keyboardRepeatOnRadioButton">
           <property name="text">
            <string>T&amp;urn on</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="keyboardRepeatOffRadioButton">
           <property name="text">
            <string>Turn o&amp;ff</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QRadioButton" name="keyboardRepeatUnchangedRadioButton">
           <property name="text">
            <string>&amp;Leave unchanged</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>159</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2">
       <widget class="QGroupBox" name="keyboardRepeatParamsGroupBox">
        <property name="title">
         <string/>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="lblDelay">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>&amp;Delay:</string>
           </property>
           <property name="buddy">
            <cstring>delay</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSlider" name="delaySlider">
           <property name="whatsThis">
            <string>If supported, this option allows you to set the delay after which a pressed key will start generating keycodes. The 'Repeat rate' option controls the frequency of these keycodes.</string>
           </property>
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>10000</number>
           </property>
           <property name="singleStep">
            <number>1000</number>
           </property>
           <property name="pageStep">
            <number>1000</number>
           </property>
           <property name="value">
            <number>5000</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickInterval">
            <number>1000</number>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QSpinBox" name="delay">
           <property name="whatsThis">
            <string>If supported, this option allows you to set the delay after which a pressed key will start generating keycodes. The 'Repeat rate' option controls the frequency of these keycodes.</string>
           </property>
           <property name="suffix">
            <string> ms</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lblRate">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>&amp;Rate:</string>
           </property>
           <property name="buddy">
            <cstring>rate</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSlider" name="rateSlider">
           <property name="whatsThis">
            <string>If supported, this option allows you to set the rate at which keycodes are generated while a key is pressed.</string>
           </property>
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>10000</number>
           </property>
           <property name="singleStep">
            <number>1000</number>
           </property>
           <property name="pageStep">
            <number>1000</number>
           </property>
           <property name="value">
            <number>5000</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickInterval">
            <number>1000</number>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QDoubleSpinBox" name="rate">
           <property name="whatsThis">
            <string>If supported, this option allows you to set the rate at which keycodes are generated while a key is pressed.</string>
           </property>
           <property name="suffix">
            <string> repeats/s</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>Test area:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="toolTip">
        <string>Allows to test keyboard repeat and click volume (just don't forget to apply the changes).</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
